source/connectors/kafka-sink-postprocessors.txt

```diff
@@ -9,10 +9,821 @@ Kafka Sink Connector Post-Processors
+:java-docs-latest:`ReplaceOptions </javadoc/com/mongodb/client/model/ReplaceOptions.html>`
+set to upsert mode.
+
+You can override the default write model by specifying a custom one in the
+``mongodb.writemodel.strategy`` configuration setting. The following
+strategies are provided with the connector:
+
+.. list-table::
+   :header-rows: 1
+   :stub-columns: 1
+
+   * - Write Model
+     - Description
+
+   * - ReplaceOneDefaultStrategy
+     - | Replaces at most one document that matches the current document by the ``_id`` field.
+       | Default value for ``writemodel.strategy`` configuration setting.
+
+   * - ReplaceOneBusinessKeyStrategy
+     - | Replaces at most one document that matches filters provided by the ``document.id.strategy`` setting.
+       | Set the following configuration: ``writemodel.strategy=com.mongodb.kafka.connect.sink.writemodel.strategy.ReplaceOneBusinessKeyStrategy``
+
+       .. seealso:: Example of usage in :ref:`writemodel-strategy-business-key`.
+
+   * - DeleteOneDefaultStrategy
+     - | Deletes at most one document that contain null value sink records that matches the id specified by the ``document.id.strategy`` setting.
+       | Implicitly specified when the configuration setting ``mongodb.delete.on.null.values=true`` is set.
+       | You can set this explicitly with the following configuration: ``writemodel.strategy=com.mongodb.kafka.connect.sink.writemodel.strategy.DeleteOneDefaultStrategy``
+
+   * - UpdateOneTimestampsStrategy
+     - | Add ``_insertedTS`` (inserted timestamp) and ``_modifiedTS`` (modified timestamp) fields into documents.
+       | Set the following configuration: ``writemodel.strategy=com.mongodb.kafka.connect.sink.writemodel.strategy.UpdateOneTimestampsStrategy``
+
+       .. seealso:: Example of usage in :ref:`writemodel-strategy-timestamps`.
+
+.. note::
+
+   Future versions of the MongoDB Kafka Connector will allow developers to
+   specify a user-created, custom strategy in the configuration settings.
+
+.. _writemodel-strategy-business-key:
+
+WriteModel Strategy: Business Keys
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+A business key is a value comprised of data within the sink record that
+identifies it as a unique document. This example shows how to define
+a business key using the data contained by multiple fields in the record as
+well as instruct the post processor to generate *BSON ObjectIds* for
+inserts, but not for updates.
+
+The following steps are required to configure this strategy:
+
+#. Create a **unique index** that corresponds to your business key in your
+   target MongoDB collection.
+#. In the connector configuration, specify the ``PartialValueStrategy``
+   as the id strategy to identify the fields that belong to the business
+   key.
+#. In the connector configuration, specify the
+   ``ReplaceOneBusinessKeyStrategy`` writemodel strategy.
+
+Suppose the parking usage data was being tracked for city blocks, defined
```

Comment:
This feels abrupt, I think because of `Suppose the` instead of `Suppose that`. I scrolled back up the page to make sure I hadn't missed parking usage data mentioned elsewhere